### ISSUE TITLE
Error returned when saving any code segment in Atom

### ISSUE THREAD

**Issue by OfficialNoms** [LordGolias/sqf#39]
Error reported when running the linter (as of the most recent push) via Atom.

Any SQF code, correct or incorrect will trigger this error.

Atom version 1.28.1

```[Linter] Error running SQFlint Error: Traceback (most recent call last):
  File "c:\users\aaron\appdata\local\programs\python\python37-32\lib\site-packages\pkg_resources\__init__.py", line 2332, in resolve
    return functools.reduce(getattr, self.attrs, module)
AttributeError: module 'sqflint' has no attribute '_main'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\aaron\AppData\Local\Programs\Python\Python37-32\Scripts\sqflint-script.py", line 11, in <module>
    load_entry_point('sqflint', 'console_scripts', 'sqflint')()
  File "c:\users\aaron\appdata\local\programs\python\python37-32\lib\site-packages\pkg_resources\__init__.py", line 480, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "c:\users\aaron\appdata\local\programs\python\python37-32\lib\site-packages\pkg_resources\__init__.py", line 2693, in load_entry_point
    return ep.load()
  File "c:\users\aaron\appdata\local\programs\python\python37-32\lib\site-packages\pkg_resources\__init__.py", line 2324, in load
    return self.resolve()
  File "c:\users\aaron\appdata\local\programs\python\python37-32\lib\site-packages\pkg_resources\__init__.py", line 2334, in resolve
    raise ImportError(str(exc))
ImportError: module 'sqflint' has no attribute '_main'
    at ChildProcess.<anonymous> (C:\Users\aaron\.atom\packages\linter-sqf\node_modules\sb-exec\lib\index.js:56:20)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)```

**Comment by OfficialNoms** [comment]
Additional: Your previous fix for the closed issue #38 works, it's your 4 most recent pushes that broke the parser. Probably something you're aware of, feel free to close this if you're current working in-progress on a few things, I've just grabbed the snapshot from the push to fix #38 and am working from that currently. 

Thanks for your continued support of this linter! It's a life saver.

**Comment by LordGolias** [comment]
@OfficialNoms, thanks for it. I hadn't noticed yet actually, so great that you found it so soon :)
